------------------------------------------------------------
/usr/bin/pip3 run on Fri Feb  8 15:18:40 2019
Downloading/unpacking pymongo-flask
  Getting page https://pypi.python.org/simple/pymongo_flask/
  Could not fetch URL https://pypi.python.org/simple/pymongo_flask/: 404 Client Error: Not Found
  Will skip URL https://pypi.python.org/simple/pymongo_flask/ when looking for download links for pymongo-flask
  Getting page https://pypi.python.org/simple/
  URLs to search for versions for pymongo-flask:
  * https://pypi.python.org/simple/pymongo_flask/
  Getting page https://pypi.python.org/simple/pymongo_flask/
  Could not fetch URL https://pypi.python.org/simple/pymongo_flask/: 404 Client Error: Not Found
  Will skip URL https://pypi.python.org/simple/pymongo_flask/ when looking for download links for pymongo-flask
  Could not find any downloads that satisfy the requirement pymongo-flask
Cleaning up...
  Removing temporary dir /tmp/pip_build_root...
No distributions at all found for pymongo-flask
Exception information:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 278, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/usr/lib/python3/dist-packages/pip/req.py", line 1178, in prepare_files
    url = finder.find_requirement(req_to_install, upgrade=self.upgrade)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 277, in find_requirement
    raise DistributionNotFound('No distributions at all found for %s' % req)
pip.exceptions.DistributionNotFound: No distributions at all found for pymongo-flask
